src/components/select/select.js

```diff
@@ -1582,4 +1582,14 @@ function SelectProvider($$interimElementProvider) {
     }
     return isScrollable;
   }
+
+}
+
+function shouldHandleKey(ev) {
+  var char = String.fromCharCode(ev.keyCode);
+  var isNonUsefulKey = (ev.keyCode <= 31);
+  var isLeftRightSelectKey = (ev.keyCode >= 91 && ev.keyCode <= 93);
+  var isFnLockKey = (ev.keyCode >= 112 && ev.keyCode <= 145);
+
```

Comment:
I'm really confused by what you mean here 😄 

You're saying we should also test for `Ctrl`, `Meta` or `Alt` and return false too?